src/codec/builtin/FixSizedTypesCodec.ts

```diff
@@ -43,6 +73,29 @@ export class FixSizedTypesCodec {
         return new Long(low, high);
     }
 
+    static encodeNumberAsLong(buffer: Buffer, offset: number, value: any): void {
+       if (value <= -TWO_PWR_63_DBL) {
```

Comment:
Do we need this check, considering that the input has to be non-negative?

Reply:
nope, removing